ui/src/components/no-code/components/tasks/Wrapper.vue

```diff
@@ -1,6 +1,6 @@
 <template>
     <slot v-if="transparent" name="tasks" />
-    <div v-else class="schema-wrapper" :class="{bordered: !merge}">
+    <div v-else class="schema-wrapper bordered" :class="{bordered: !merge}">
```

Comment:
```suggestion
    <div v-else class="schema-wrapper bordered">
```

With your change, the `merge` prop does not have any effect anymore.

Is this intended?